src/index.html

```diff
@@ -20,8 +21,437 @@
+        alt="logo"
+      />
 
-  <body>
-    <h1>Catalog</h1>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="is-active nav__link"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#samsung"
+              class="nav__link"
+            >
+              SAMSUNG
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#smartphones"
+              class="nav__link"
+            >
+              SMARTPHONES
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              class="nav__link"
+              href="#laptops & computers"
+              data-qa="nav-hover"
+            >
+              LAPTOPS & COMPUTERS
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#gadgets"
+              class="nav__link"
+            >
+              GADGETS
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#tablets"
+              class="nav__link"
+            >
+              TABLETS
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#photo"
+              class="nav__link"
+            >
+              PHOTO
+            </a>
+          </li>
+          <li class="nav__list-item">
+            <a
+              href="#video"
+              class="nav__link"
+            >
+              VIDEO
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="container">
+      <section
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="product-photo"
+          class="card__photo"
+        />
+
+        <div class="card__top">
+          <h3 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+
+          <p class="card__code">Product code: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__rating-response">Reviews: 5</p>
+        </div>
+
+        <p class="card__price">
+          <span class="card__price-inscription">Price:</span>
+
+          <span class="card__price-count">$2,199</span>
+        </p>
+
+        <div class="card-button">
+          <a
+            href="#"
+            class="card-button__link"
+            data-qa="card-hover"
```

Comment:
The `data-qa` attribute value 'card-hover' is used multiple times in the document, which is not ideal for a unique identifier. Consider using unique values if they are meant to be used as unique hooks for QA or testing purposes.